pkgs/servers/search/manticoresearch/default.nix

```diff
@@ -0,0 +1,90 @@
+{ lib, stdenv, fetchFromGitHub, fetchurl
+, bison, cmake, flex
+, boost, icu, mariadb-connector-c, re2
+}:
+let
+  boost_static = boost.override { enableStatic = true; };
+  columnar = stdenv.mkDerivation rec {
```

Comment:
shouldn't we package it separately?

Reply:
You mean in a separate pkgs/servers/search/manticoresearch/columnar.nix file? Or an entire new top-level package?

Reply:
Hm not sure. If this dependency is only used in manticore we could leave it like this